src/Parser/Rogue/Subtlety/Modules/Features/Abilities.js

```diff
@@ -10,26 +11,90 @@ class Abilities extends CoreAbilities {
       spell: SPELLS.VANISH,
       category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
       getCooldown: haste => 90,
-      recommendedCastEfficiency: 1.0,
+      //Vanish is often delayed.
+      recommendedCastEfficiency: 0.90,
     },
```

Comment:
Adding a bit more annoying comments here: You sure you don't want anything here in the suggestion about it often being delayed?

Reply:
Vanish usage depends a on legionaries and talents, for now I'm trying to get all of the general features in. Putting a line like "try not to delay it too much" doesn't seem helpful to me.

Reply:
It's helpful until you've implemented those things, and if the legendaries do get implemented will they override this, or how will that be handled? If they don't, it should some sort of suggestion regardless.
A general rule of thumb as an extraSuggestion or the likes is generally implemented and sounds helpful to me, if that is possible. 
I understand you want to get the general features implemented, but telling people they're not casting X spell enough without giving them a reason to as to why they're casting it too little or when they should be casting it more, can just lead to confusion or incorrect play in their hunt to use it more.

Reply:
Ok agreed, no suggestions for now.